### DISCUSSION TITLE
'Keep Four Alive' Rewarded when only 2 snakes survived to the turn limit

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#90] [Bug Reports]
I got the reward for this challenge with this game: https://play.battlesnake.com/g/6ab26b84-c5db-4feb-aa95-5590085680b6/?title=Keep+Four+Alive

But only three of my snakes actually made it to turn 600, the other dies pretty as early as turn 100ish

I think this is a bug right?

**Comment by coreyja** [comment]
I couldn't help myself lol, ran an "Eight Snake Chaos" Game and the same thing happened. I got the badge but some of my snakes died well before the 300 turns
https://play.battlesnake.com/g/11949434-f0be-4e40-984c-fc788959531a/?title=Eight+Snake+Chaos

**Comment by bvanvugt** [comment]
@coreyja thanks for reporting this - we've removed these challenges, for the time being. You're correct in that they were being scored incorrectly. 

We plan to re-release these challenges in the future, possibly under new names and new win conditions. Stay tuned.

Gonna close this as shipped, and we'll announce the new stuff once it's live.
